pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -67,6 +69,9 @@ var (
 	LocalCertDir = env.RegisterStringVar("ROOT_CA_DIR", "./etc/cacerts",
 		"Location of a local or mounted CA root")
 
+	useRemoteCerts = env.RegisterBoolVar("USE_REMOTE_CERTS", false,
+		"Whether to try to load remote CA certs.")
```

Comment:
Please give more concrete explanation. It's supposed to load remote CA certs from external K8s cluster, right?